src/components/identifier/identifier.module.scss

```diff
@@ -6,14 +6,14 @@
 .identifier {
   text-decoration: none;
   color: currentColor;
-  font-size: 18px;
+  font-size: 0px;
   font-weight: 500;
   font-family: "Work Sans", sans-serif;
 }
 
 .image {
-  width: 26px;
-  height: 30px;
-  margin-right: -5px;
-  margin-bottom: 4px;
+  //width: 26px;
+  height: 50px;
+  //margin-right: -5px;
```

Comment:
I would make sure that these comments are removed in a further update to make sure we are cleaning up code correctly